<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionPopupWindow</class>
 <widget class="QDialog" name="ConnectionPopupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="independent">
       <property name="text">
        <string>independent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="allServerCPUHigherThan">
       <property name="text">
        <string>allServerCPUHigherThan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="allServerCPUHigherThanValue"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>(%)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="anyServerCPUHigherThan">
       <property name="text">
        <string>anyServerCPUHigherThan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>(%)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="anyServerCPUHigherThanValue"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="runPerformanceJobGroupAfter">
       <property name="text">
        <string>runPerformanceJobGroupAfter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>(minutes)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="runPerformanceJobGroupAfterValue"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="runPerformanceJobGroupInEvenRun">
       <property name="text">
        <string>runPerformanceJobGroupInEvenRun</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OkBtn">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
